mister.py

```diff
@@ -94,6 +94,9 @@ def handleRunBat(gGator):
                 if cmdline not in handled:
                     handled[cmdline] = convertImgMount(cmdline, gGator)
                 runFileClone.write(handled[cmdline] + '\n')
+            elif cmdline.lower().startswith("config "):
+                converted_sound_command = convertSoundConfig(cmdline)
+                runFileClone.write(converted_sound_command + '\n')
```

Comment:
you can do it in one line I guess ;) : `runFileClone.write(convertSoundConfig(cmdline) + '\n')`

Reply:
While that is true. I chose to not do so because I thought later modifications would be easier if it was already split into two lines. I also think it increases readability.

Of course this is your repository and I will comply with your wishes if you insist.